### PR TITLE
add global flag for enable wasmjit verifier

### PR DESCRIPTION
**Pull request by carltraveler**

### CODE REVIEW

File: common/config/config.go [hunk 1]
```diff
@@ -35,6 +35,16 @@ import (
 
 var Version = "" //Set value when build project
 
+type VerifyMethod int
+
+const (
```
Comment (lucas7788) [inline common/config/config.go:40]: const(
        NoneVerifyMethod   VerifyMethod = iota
	InterpVerifyMethod 
	JitVerifyMethod    
)

File: common/config/config.go [hunk 2]
```diff
@@ -35,6 +35,16 @@ import (
...
+	NoneVerifyMethod   = VerifyMethod(0)
+	InterpVerifyMethod = VerifyMethod(1)
+	JitVerifyMethod    = VerifyMethod(2)
+)
+
+var WasmVerifyMethod = InterpVerifyMethod
```
Comment (laizy) [inline common/config/config.go]: this should be removed. the default is already handled by `WasmVerifyMethodFlag`

File: main.go [hunk 3]
```diff
@@ -209,6 +211,17 @@ func initLog(ctx *cli.Context) {
 	}
 }
 
+func initGlobalParam(ctx *cli.Context) {
+	enableWasmJitVerify := ctx.GlobalBool(utils.GetFlagName(utils.WasmVerifyMethodFlag))
```
Comment (laizy) [inline main.go]: there are already other global params already I think, move this logic to there?

File: cmd/utils/flags.go [hunk 4]
```diff
@@ -52,6 +52,10 @@ var (
 		Name:  "disable-event-log",
 		Usage: "Discard event log output by smart contract execution",
 	}
+	WasmVerifyMethodFlag = cli.BoolFlag{
```
Comment (lucas7788) [inline cmd/utils/flags.go:55]: you should add WasmVerifyMethodFlag to AppHelpFlagGroups in usage.go

File: common/config/config.go [hunk 5]
```diff
@@ -38,12 +38,12 @@ var Version = "" //Set value when build project
...
-	NoneVerifyMethod   = VerifyMethod(0)
-	InterpVerifyMethod = VerifyMethod(1)
-	JitVerifyMethod    = VerifyMethod(2)
+	NoneVerifyMethod VerifyMethod = iota
+	InterpVerifyMethod
+	JitVerifyMethod
 )
 
-var WasmVerifyMethod = InterpVerifyMethod
+var WasmVerifyMethod VerifyMethod
```
Comment (laizy) [inline common/config/config.go]: I think you need delete this.

File: common/config/config.go [hunk 6]
```diff
@@ -38,12 +38,12 @@ var Version = "" //Set value when build project
 type VerifyMethod int
 
 const (
-	NoneVerifyMethod   = VerifyMethod(0)
-	InterpVerifyMethod = VerifyMethod(1)
-	JitVerifyMethod    = VerifyMethod(2)
+	NoneVerifyMethod VerifyMethod = iota
+	InterpVerifyMethod
+	JitVerifyMethod
```
Comment (laizy) [inline common/config/config.go:42]: reorder None to the last.

**Review by laizy — APPROVED** [review]

**Pull request merged**